packages/push/tests/float.rs

```diff
@@ -1,7 +1,16 @@
 #![cfg(test)]
-#![allow(clippy::unwrap_used)]
-#![allow(clippy::tuple_array_conversions)]
-#![allow(clippy::arithmetic_side_effects)]
+// TODO: since inner attributes are unstable, we can't use rustversion here.
+// Once we revert this commit, this is proper again.
+#![allow(clippy::allow_attributes_without_reason)]
+#![allow(
+    clippy::unwrap_used,
+    // reason = ".unwrap() is the best way to deal with errors in integration tests"
+)]
+#![allow(
+    clippy::arithmetic_side_effects,
+    // reason = "The tradeoff safety <> ease of writing arguably lies on the ease of writing side \
+    //           for test code."
+)]
```

Comment:
Again, this is different than the others, and I"m not sure why.

Reply:
see above ^^